picard/ui/edittagdialog.py

```diff
@@ -514,30 +514,18 @@ def _modified_tag(self):
             list(self.metadata_box.tag_diff.new[self.tag]) or [""]
         )
 
-    def _cleanup_modified_tags(self):
-        """Remove empty values from modified tags."""
+    def _modified_tags(self):
```

Comment:
The naming is a bit unclear I think. Calling this `_modified_tag` is very similar to `_modified_tags`, but they do behave differently. While one gives you all the values for the currently active tag for editing, the other gives you all tags to be saved.

Not sure how to best rename them. Maybe the original `_modified_tag` should be renamed to something like `_current_tag` or `_current_tag_values`.

Reply:
For now, I'll renamed it to _modified_tags_without_empty_values() so the patch doesn't change lines you're working on, but I agree we should rethink names in this file.